packages/seal-meilisearch-adapter/src/MeilisearchSearcher.php

```diff
@@ -143,6 +151,7 @@ private function recursiveResolveFilterConditions(Index $index, array $condition
                 $filter instanceof Condition\SearchCondition => $query = $filter->query,
                 $filter instanceof Condition\EqualCondition => $filters[] = $filter->field . ' = ' . $this->escapeFilterValue($filter->value),
                 $filter instanceof Condition\NotEqualCondition => $filters[] = $filter->field . ' != ' . $this->escapeFilterValue($filter->value),
+                $filter instanceof Condition\NotInCondition => $filters[] = $filter->field . ' NOT IN [' . $this->escapeArrayFilterValues($filter->values) . ']',
```

Comment:
Syntax specicially for Meilisearch for `IN` and `NOT IN` according to docs: https://www.meilisearch.com/docs/learn/filtering_and_sorting/filter_expression_reference#in.

Could also use the `$filter->createAndCondition()` instead. Not sure what's preferred, the generic way or the one more closely to the adapter.